test/e2e/facility-helpers.js

```diff
@@ -0,0 +1,267 @@
+              sl1: [
+                'MentalHealthCare'
+              ],
+              sl2: []
+            },
+            {
+              sl1: [
+                'PrimaryCare'
+              ],
+              sl2: []
+            }
+          ]
+        },
+        feedback: {
+          health: {
+            effective_date_range: 'Jun 2016 - Nov 2016',
+            primary_care_routine: 0.73,
+            primary_care_urgent: 0.74,
+            specialty_care_routine: null,
+            specialty_care_urgent: null
+          }
+        },
+        access: {
+          health: {
+            primary_care_wait_days: 5.977272,
+            primary_care_wait_sample_size: 44,
+            specialty_care_wait_days: 4.327272,
+            specialty_care_wait_sample_size: 110,
+            mental_health_wait_days: 3.962962,
+            mental_health_wait_sample_size: 27,
+            urgent_consult_percentage: null,
+            urgent_consult_sample_size: null
+          }
+        }
+      }
+    },
+    {
+      id: 'vba_343o',
+      type: 'va_facilities',
+      attributes: {
+        unique_id: '343o',
+        name: 'Los Angeles Ambulatory Care Center',
+        facility_type: 'va_benefits_facility',
+        classification: 'OUTBASED',
+        website: null,
+        lat: 34.05171315,
+        'long': -118.2387403,
+        address: {
+          physical: {
+            address_1: '351 East Temple Street',
+            address_2: 'Room A-444',
+            address_3: null,
+            city: 'Los Angeles',
+            state: 'CA',
+            zip: '90012'
+          },
+          mailing: {}
+        },
+        phone: {
+          main: '213-253-2677 Ext 24759',
+          fax: ''
+        },
+        hours: {
+          monday: '8:00AM-2:30PM',
+          tuesday: '8:00AM-2:30PM',
+          wednesday: '8:00AM-2:30PM',
+          thursday: '8:00AM-2:30PM',
+          friday: 'By Appointment Only',
+          saturday: 'Closed',
+          sunday: 'Closed'
+        },
+        services: {
+          benefits: {
+            other: '',
+            standard: []
+          }
+        },
+        feedback: {},
+        access: {}
+      }
+    },
+  ],
+  links: {
+    self: 'http://www.example.com/v0/facilities/va?bbox%5B%5D=-118.53149414062501&bbox%5B%5D=33.91487347147951&bbox%5B%5D=-118.07762145996095&bbox%5B%5D=34.199308935560154&page=1',
+    first: 'http://www.example.com/v0/facilities/va?bbox%5B%5D=-118.53149414062501&bbox%5B%5D=33.91487347147951&bbox%5B%5D=-118.07762145996095&bbox%5B%5D=34.199308935560154&page=1&per_page=20',
+    prev: null,
+    next: null,
+    last: 'http://www.example.com/v0/facilities/va?bbox%5B%5D=-118.53149414062501&bbox%5B%5D=33.91487347147951&bbox%5B%5D=-118.07762145996095&bbox%5B%5D=34.199308935560154&page=1&per_page=20'
+  },
+  meta: {
+    pagination: {
+      current_page: 1,
+      per_page: 20,
+      total_pages: 1,
+      total_entries: 3
+    }
+  }
+};
+
+const facilityData = {
```

Comment:
This looks equivalent to `resultsData.data[0]`. Would it be more succinct to set it to a deep copy of that?